src/App.js

```diff
@@ -1,18 +1,34 @@
 import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import { isLoggedIn } from './redux/slices/auth';
+import Login from './Pages/Auth/Login';
+import Register from './Pages/Auth/Register';
+import Home from './Pages/Home/Home';
+import './App.css';
+import Layout from './Layouts/Layout';
 
 function App() {
   const dispatch = useDispatch();
-  const User = useSelector((state) => state.user);
+  const user = useSelector((state) => state.user);
   useEffect(() => {
     dispatch(isLoggedIn());
   }, []);
   return (
     <>
-      HELLO
-      {' '}
-      {User && User.first_name}
+      <BrowserRouter>
+        <Layout>
+          <Routes>
+            <Route exact path="/" element={<Home />} />
+            {!user && (
+            <>
+              <Route path="/login" element={<Login />} />
+              <Route path="/register" element={<Register />} />
+            </>
+            )}
+          </Routes>
```

Comment:
[Optional] What's your opinion about extracting all of the routes to a separate component? So that we can keep our App component as small as possible.